src/Symfony/Component/ExpressionLanguage/Tests/ParserTest.php

```diff
@@ -161,4 +161,44 @@ private function createGetAttrNode($node, $item, $type)
     {
         return new Node\GetAttrNode($node, new Node\ConstantNode($item), new Node\ArgumentsNode(), $type);
     }
+
+    /**
+     * @expectedException \Symfony\Component\ExpressionLanguage\SyntaxError
+     */
+    public function testParseWithStringAsInvalidName()
+    {
+        $lexer = new Lexer();
+        $parser = new Parser(array());
+        $parser->parse($lexer->tokenize('foo."#"'), array('foo'));
+    }
```

Comment:
Checked the travis build log and found the following error:
```bash
The command "sh -c 'if [ "$components" = "no" ]; then sh -c "ls -d src/Symfony/*/* | parallel --gnu --keep-order '\''echo \"Running {} tests\"; phpunit --exclude-group tty,benchmark {};'\''"; fi;'" exited with 1.
```
So I guess it can be ignored.